charts/pega/values-minimal.yaml

```diff
@@ -95,6 +95,8 @@ global:
 
   # Specify the Pega tiers to deploy
   # For a minimal deployment, use a single tier to reduce resource consumption.
+  # Note: The nodeType Stream is deprecated, please remove it and enable External Kafka
+  # configuration under External Services
   tier:
     - name: "minikube"
       nodeType: "Stream,BackgroundProcessing,WebUser,Search"
```

Comment:
In the minikube deployment, nodeType would always contain Stream. I guess, with `stream.enabled` to true, nodeType should no more contain `Stream`. Can you please take care?

Reply:
stream.enable is not enabled by default so that's why nodeType still has stream

Reply:
Yes. But if stream.enable is set, nodetype should not have stream. I could not find those changes.